scripts/tslint/rules/noInOperatorRule.ts

```diff
@@ -12,7 +12,7 @@ export class Rule extends Lint.Rules.AbstractRule {
 function walk(ctx: Lint.WalkContext<void>): void {
     ts.forEachChild(ctx.sourceFile, recur);
     function recur(node: ts.Node): void {
-        if (node.kind === ts.SyntaxKind.InKeyword && node.parent.kind === ts.SyntaxKind.BinaryExpression) {
+        if (node.kind === ts.SyntaxKind.InKeyword && node.parent!.kind === ts.SyntaxKind.BinaryExpression) {

```

Comment:
parent is not optional, why is this required?

Reply:
This uses our published `typescript.d.ts`, not the one in this PR -- so if we merge this and the newest published version has `parent` non-optional, we'll have to change it then.